apps/web/src/components/TrendingSuggestionList/TrendingSuggestionList.utils.tsx

```diff
@@ -1,17 +1,10 @@
-import Icon from '../Icon/Icon';
-import { ReactComponent as ReactIcon } from '../../assets/reactSubRedditIcon.svg';
-
 export const metaDataItems = [
```

Comment:
You could consolidate these items into a `metaDataItem` since they are identical and then use that variable 3 times in the `metaDataItems` array. Then you would only need to make one change. Since this is just mock data though, I don't have strong feelings about doing this, just thought I would point it out.

Reply:
That's a good point. I thought about doing that as well, but kept getting an error when testing the component. The error was caused by having multiple elements with the same output in the heading property, but I recently learned I could have used the "allBy" query to fix that.